### ISSUE TITLE
Archetype 1.8 Label Templates & Fieldsets Sorting Bugs

### ISSUE THREAD

**Issue by ane-ta** [imulus/Archetype#278]
After updating to Archetype 1.8
(umbraco 7.2.1)
1. Label templates containing multiple {{field}} macros stopped working: showing just empty string.
![image](https://cloud.githubusercontent.com/assets/6615595/8203790/c284fb5a-14fd-11e5-99ab-296fbd552a92.png)
![image](https://cloud.githubusercontent.com/assets/6615595/8203799/ca19ea10-14fd-11e5-914d-55b202bfffab.png)

==================================================
2. If one of fieldsets contains richtext editor the fieldset sorting fails:
This is the list of fieldsets
![image](https://cloud.githubusercontent.com/assets/6615595/8203826/f6441e4e-14fd-11e5-8ce6-2d699697a8c3.png)

Person words fieldset contains richtext editor:
![image](https://cloud.githubusercontent.com/assets/6615595/8203838/0b0aa122-14fe-11e5-8f47-2c82625fedeb.png)

When dragging sort icon I get a light blue area expanding below and javascript errors of one type
![image](https://cloud.githubusercontent.com/assets/6615595/8203871/47a5b82e-14fe-11e5-9527-6e4f43e3067d.png)

![image](https://cloud.githubusercontent.com/assets/6615595/8203882/57b8b680-14fe-11e5-8346-cfcbc22d7c4c.png)

If I delete the fieldset with richtext editor the bug disappears.

**Comment by kgiszewski** [comment]
Hello and thank you for using Archetype.

Your issue #1 was already reported and fixed here: https://github.com/imulus/Archetype/issues/276 The next patch release will address it.

Your issue #2 is interesting.  I suddenly started getting the blue box as well during development and had to update the ui-sortable options here: https://github.com/imulus/Archetype/commit/34b4209de7000ffa0e164d7bf8f0b1bff6eeb7f3

You are on a few patch versions older of Umbraco and I'm curious if the RTE changed between 7.2.1 and 7.2.6.

Anyway...

Your options to get going right now:

1. Downgrade to 1.7.5 or keep 1.8 and...
2. Patch the labelService with the code referenced in the other issue
3. Revert the ui-sortable code to fix the RTE (possibly) though you should ensure that your cache is cleared and all that.

**Comment by kgiszewski** [comment]
FYI, the only issue I am experiencing with 1.8 is the multiple label issue and not the RTE issue.

**Comment by ane-ta** [comment]
Hello kgiszewski
Thank you for a quick reply.

Reverting 3 point caused returning another bug: when sorting fieldsets with rte editors these editors become empty and disabled.
So this was not an option.

But I've found out what was the problem: the error was about unexisting function in Underscore.js package. After updating Undescore.js to last version (1.8.1 instead of 1.4.1) the problem was solved.

So I believe current umbraco has this package updated too.
